scripts/include-sf.js

```diff
@@ -8,7 +8,12 @@ const npmShow = JSON.parse(shelljs.exec('npm show @salesforce/cli dist-tags --js
 // make sure latest-rc tag exists, if not fallback to latest
 const tag = npmShow['latest-rc'] ? 'latest-rc' : 'latest';
 console.log(`---- Installing  @salesforce/cli@${tag} ----`);
-shelljs.exec(`npm install @salesforce/cli@${tag} -g`);
+// if latest/rc has moved to v2 already, we don't want to bundle that.  We'll use the highest v1 version available
+if (npmShow[tag].startsWith('1')) {
+  shelljs.exec(`npm install @salesforce/cli@${tag} -g`);
+} else {
+  shelljs.exec(`npm install @salesforce/cli@^1 -g`);
+}
```

Comment:
I think we should always install `^1` here too. Otherwise there could be situations where a Docker container includes a different version of `sf`. I think it's better to keep all bundled `sf` versions the same and accept that they might be a bit ahead for these last few releases